config.toml

```diff
@@ -183,10 +183,7 @@ enable = true
     pre = "<i class='fab fa-github'></i>"
     # post = "<span class='alert'>New!</span>" 
 
-[[menu.main]]
-    name = "Get Started"
-    weight = 70
-    url = "docs/get-started/"
+
```

Comment:
This removes the change @aannttoo put in with PR #50 - is that intentional?

Reply:
Yes because the new button includes what @aannttoo Has done here.